modules/lightning_features/lightning_media/src/Plugin/EntityBrowser/Widget/EntityFormProxy.php

```diff
@@ -177,6 +180,24 @@ protected function getInputValue(FormStateInterface $form_state) {
   }
 
   /**
+   * Returns an array of bundles from the entity browser's widget context.
+   *
+   * @param \Drupal\Core\Form\FormStateInterface $form_state
+   *   The current form state.
+   *
+   * @return array
+   *   The target media bundles.
+   */
+  protected function getTargetBundles(FormStateInterface $form_state) {
+    $bundles = [];
```

Comment:
@phenaproxima ,

I think so. Will make that change when I get a second or two. Thanks.